src/ui/units/dash/containers/App/App.tsx

```diff
@@ -85,6 +89,7 @@ export function App({...routeProps}: RouteComponentProps) {
                 return;
             }
 
+            sendEmbedDashHeight(wrapRef);
```

Comment:
Let's add condition of checking embedded mode into sendEmbedDashHeight function

Reply:
It shouldn't be here. I forgot to remove